test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1168,33 +1168,80 @@ def _revert_pbsconf_server(self, vals_to_set):
                 if restart_pbs:
                     # Restart all
                     server.pi.restart(server.hostname)
-                    if not server.isUp():
-                        self.fail("Server is not up")
-                    if not self.mom.isUp():
-                        self.fail("Mom is not up")
-                    if not self.scheduler.isUp():
-                        self.fail("Scheduler is not up")
-                    if not self.comm.isUp():
-                        self.fail("comm is not up")
-
+                    rv = server.isUp()
+                    if not rv:
+                        self.logger.error('Server '
+                                          + comm.hostname
+                                          + ' is down')
+                        msg = 'Failed to restart server '
+                        msg += server.hostname
+                        self.assertTrue(rv, msg)
```

Comment:
Instead of this, please use self.fail() instead, same comment for other instances